tests/test_analog.py

```diff
@@ -292,12 +292,14 @@ def test_hamevo_endianness_cnot() -> None:
     assert torch.allclose(wf_cnot, wf_hamevo, rtol=RTOL, atol=ATOL)
 
 
-@pytest.mark.parametrize("duration", [torch.rand(1), "duration"])
+@pytest.mark.parametrize("duration", [torch.rand(1)])
+@pytest.mark.parametrize("batch_size", [1, 5])
 @pytest.mark.parametrize("ode_solver", [SolverType.DP5_SE, SolverType.KRYLOV_SE])
 def test_timedependent(
     tparam: str,
     param_y: float,
     duration: float,
+    batch_size: int,
     n_steps: int,
     torch_hamiltonian: Callable,
     hamevo_generator: Sequence,
```

Comment:
Can you also parametrize over the `use_sparse = True / False`?

Reply:
sure